schemas/iso19139/src/main/plugin/iso19139/index-fields/default.xsl

```diff
@@ -503,10 +503,14 @@
 
       <!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
 
-      <xsl:for-each
```

Comment:
In Data Identification the cardinality is [1..*], with the change will be indexed only the first language. Probably it's the most usual case, but would be better to manage all languages defined.